SteamBotUnitTest/SteamTrade/GenericInventoryTest.cs

```diff
@@ -0,0 +1,25 @@
+﻿using NUnit.Framework;
+using SteamKit2;
+using SteamTrade;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SteamBotUnitTest.SteamTrade
+{
+    [TestFixture]
+    class GenericInventoryTest
+    {
+        [Test]
+        public void LoadTest()
+        {
+            var inventory = new GenericInventory(new SteamWeb());
+            //76561198104350201 is my friend's Steam account with lots of Dota2 items which will test the start/more way of loading. It's inventory will remain public and contain more than 2000 items long term.
```

Comment:
Most importantly, unit tests are not supposed to test whether an external API (it's not even an official API btw) works or not, but whether the changes in our code haven't managed to break existing functionality. In this case, network mocks would be necessary.